src/ds3_string_multimap_impl.h

```diff
@@ -0,0 +1,36 @@
+/*
+* ******************************************************************************
+*   Copyright 2014-2015 Spectra Logic Corporation. All Rights Reserved.
+*   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+*   this file except in compliance with the License. A copy of the License is located at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+*   or in the "license" file accompanying this file.
+*   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+*   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+*   specific language governing permissions and limitations under the License.
+* ****************************************************************************
+*/
+
+
+#ifndef __DS3_STRING_MULTIMAP_IMPL__
+#define __DS3_STRING_MULTIMAP_IMPL__
+
+#include <glib.h>
+
```

Comment:
Probably should have
#ifdef __cplusplus
extern "C" {
#endif

in all of our header files, otherwise it causes problems when we unit test them, since the unit tests compile in C++.